src/main/java/com/epam/aidial/cfg/web/controller/ToolSetController.java

```diff
@@ -70,4 +72,10 @@ public ToolSetDto getSnapshot(@PathVariable String toolSetName, @PathVariable In
     public Collection<ToolSetDto> getAllAtRevision(@PathVariable Integer revision) throws Exception {
         return toolSetFacade.getAllAtRevision(revision);
     }
+
+    @GetMapping(path = "/{toolSetName}/tools", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
i'd expect to see autodetected nature of this tools un url. just `/tool` looks non obvious

Reply:
Re-named to /discovered-tools